shared/dataflow/codeql/dataflow/TaintTracking.qll

```diff
@@ -0,0 +1,83 @@
+/**
+ * Provides classes for performing local (intra-procedural) and
+ * global (inter-procedural) taint-tracking analyses.
+ */
+
+import DataFlow
+import DataFlowImpl
+import DataFlowParameter
```

Comment:
The `DataFlowImpl` import should definitely be private, and the others should possibly also be private.

Reply:
Fixed.